src/www/firewall_rules.php

```diff
@@ -422,6 +422,9 @@
                   <tr ondblclick="document.location='firewall_rules_edit.php?id=<?=$i;?>';">
                     <td>
                       <input type="checkbox" name="rule[]" value="<?=$i;?>"  />
+<?php                 if ($selected_if != "FloatingRules" && isset($filterent['floating']) && $filterent['floating'] === 'yes'): ?>
+                      <i class="fa fa-random" data-toggle="tooltip" title="<?= gettext('floating rule') ?>"></i>
+<?php                 endif; ?>
```

Comment:
We should make a second line for the icon key table, split the display in half and add a floating icon on/off line like all the other icons do.